src/azure-cli/azure/cli/command_modules/containerapp/_params.py

```diff
@@ -160,6 +160,10 @@ def load_arguments(self, _):
         c.argument('instrumentation_key', options_list=['--dapr-instrumentation-key'], help='Application Insights instrumentation key used by Dapr to export Service to Service communication telemetry')
         c.argument('dapr_connection_string', options_list=['--dapr-connection-string', '-d'], help='Application Insights connection string used by Dapr to export service to service communication telemetry.')
 
+    with self.argument_context('containerapp env update', arg_group='Dapr') as c:
+        c.argument('instrumentation_key', options_list=['--dapr-instrumentation-key'], help='Application Insights instrumentation key used by Dapr to export Service to Service communication telemetry. Use "none" to remove it.')
```

Comment:
why do we need to add --dapr-instrumentation-key?

Reply:
Updated, we should not add --dapr-instrumentation-key, also, we need to hide it because this argument is in deprecate process